spec/controllers/invites_controller_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe InvitesController, type: :controller do
+end
```

Comment:
@dcuadraq it was created by the generator but I didn't remove it because maybe in a future we're going to create tests on that controller 🤓